CHANGELOG.md

```diff
@@ -15,6 +15,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   `VcpuExit::IoOut`. The average for these VM exits is not emitted since
   it can be deduced from the available emitted metrics.
 
+### Deprecated
+
+- Firecracker's `--start-time-cpu-us` parameter is deprecated and will be
```

Comment:
should we also add a deprecation print to highlight when someone uses this parameter?

Reply:
It is always use by the jailer so it would always get printed but customers wouldn't have a way to supress it.

Reply:
I mean if customers use Firecracker without our jailer then they can notice the new deprecation print.
For customers using our jailer, yes they wouldn't be able to suppress the print but would having an extra print at the start be an issue for them?

Reply:
> It is always use by the jailer so it would always get printed but customers wouldn't have a way to suppress it.

If the jailer stopped using it, would that be a customer visible change?

Reply:
Not as far as I can see. The only thing this affects is the `api_server.process_startup_time_cpu_us` metric and it will continue to have the same value.

Reply:
Could we then not use it anymore from the jailer, and add the deprecation warning? That way, customers would only see the deprecation warning if they explicitly use the flag.

Reply:
it makes sense to to stop using it in the jailer as well since we are deprecating it.

Reply:
Done